cfme/utils/appliance/__init__.py

```diff
@@ -1387,11 +1387,11 @@ def read_repos(self):
         name_regexp = re.compile(r"^\[update-([^\]]+)\]")
         baseurl_regexp = re.compile(r"baseurl\s*=\s*([^\s]+)")
         for repofile in self.get_repofile_list():
-            result = self.ssh_client.run_command("cat /etc/yum.repos.d/{}".format(repofile))
-            if result.failed:
+            ssh_result = self.ssh_client.run_command("cat /etc/yum.repos.d/{}".format(repofile))
+            if ssh_result.failed:
                 # Something happened meanwhile?
                 continue
-            out = result.output.strip()
+            out = ssh_result.output.strip()
```

Comment:
I don't think this fixes the exception quoted in the description. The failure was from L1401, where we're modifying the SSHResult object by setting `result[<some key>] = <thing>`

Reply:
Talked offline, changing result to a different name.